File: CS50's X/WEEK 6/sentimental-cash/cash.py
from cs50 import get_float


def main():

    coins = 0

    while True:
        change = get_float("Change owed: ")
        if change > 0:
            change = 100 * change
            break

    while (change >= 25):
        coins += 1
        change -= 25

    while (change >= 10):
        coins += 1
        change -= 10

    while (change >= 5):
        coins += 1
        change -= 5

    while (change >= 1):
        coins += 1
        change -= 1

    print(f"{coins}")


main()

File: CS50's X/WEEK 1/cash/cash.c
#include <cs50.h>
#include <stdio.h>

void print_row(int spaces, int briks);

int main(void)
{
    int change;
    do
    {
        change = get_int("Change owed: ");
    }
    while (change < 1);

    int coins = 0;

    while (change >= 25)
    {
        coins++;
        change -= 25;
    }

    while (change >= 10)
    {
        coins++;
        change -= 10;
    }

    while (change >= 5)
    {
        coins++;
        change -= 5;
    }

    while (change >= 1)
    {
        coins++;
        change--;
    }

    printf("%i\n", coins);
}
